requirements.txt

```diff
@@ -1 +1,7 @@
-configobj>=5.0.2
\ No newline at end of file
+configobj>=5.0.2
+#xmlrpc
```

Comment:
I was young and dumb when I created this set of requirements. You can remove anything that's commented out and please specify versions here for safety's sake.

Reply:
Good stuff :-) Opt. `--system` for me it's the best 
I can developing/releasing/verifing independently from my webfaction's account, by my own deployment tool written in Python.